crates/core/migrations/2022-05-19-200746_create_instruction_tables/up.sql

```diff
@@ -0,0 +1,118 @@
+create table buy_instructions (
```

Comment:
@imabdulbasit there are 2 version of the buy instruction.  This is the private buy where the offer is locked to the current owner. What we are using is the publc_buy where the offer is on the NFT and doesn't become void when the nft changes ownership.

We should index both instructions with the public_buy_instruction hopefully being the one used more by other marketplaces.

Reply:
Yes, I know. I'll index the public_buy too